runtime/rococo/src/bridge_messages.rs

```diff
@@ -0,0 +1,409 @@
+	pub type FromRococoEncodedCall = messages_target::FromBridgedChainEncodedMessageCall<crate::Call>;
+
+	/// Call-dispatch based message dispatch for Rococo -> Wococo messages.
+	pub type FromRococoMessageDispatch = messages_target::FromBridgedChainMessageDispatch<
+		AtWococoWithRococoMessageBridge,
+		crate::Runtime,
+		pallet_balances::Pallet<crate::Runtime>,
+		crate::AtWococoFromRococoMessagesDispatch,
+	>;
+}
+
+#[cfg(test)]
+mod tests {
+	use bridge_runtime_common::messages;
+	use parity_scale_codec::Encode;
+	use super::*;
+
+	#[test]
+	fn ensure_rococo_messages_weights_are_correct() {
+		// **NOTE**: the main purpose of this test is to be sure that any message that is sumbitted
+		// to (any) inbound lane in Rococo<>Wococo bridge can be delivered to the bridged chain.
+		// Since we deal with testnets here, in case of failure + urgency:
+		//
+		// 1) ping bridges team about this failure;
+		// 2) comment/remove the test.
+
+		// we don't have any knowledge of messages-at-Rococo weights, so we'll be using
+		// weights of one of our testnets, which should be accurate enough
+		type Weights = pallet_bridge_messages::weights::RialtoWeight<crate::Runtime>;
+
+		pallet_bridge_messages::ensure_weights_are_correct::<Weights>(
+			DEFAULT_MESSAGE_DELIVERY_TX_WEIGHT,
+			ADDITIONAL_MESSAGE_BYTE_DELIVERY_WEIGHT,
+			MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT,
+			PAY_INBOUND_DISPATCH_FEE_WEIGHT,
+		);
+
+		let max_incoming_message_proof_size = bp_rococo::EXTRA_STORAGE_PROOF_SIZE.saturating_add(
+			messages::target::maximal_incoming_message_size(bp_rococo::max_extrinsic_size()),
+		);
+		pallet_bridge_messages::ensure_able_to_receive_message::<Weights>(
+			bp_rococo::max_extrinsic_size(),
+			bp_rococo::max_extrinsic_weight(),
+			max_incoming_message_proof_size,
+			messages::target::maximal_incoming_message_dispatch_weight(bp_rococo::max_extrinsic_weight()),
+		);
+
+		let max_incoming_inbound_lane_data_proof_size = bp_messages::InboundLaneData::<()>::encoded_size_hint(
+			bp_rococo::MAXIMAL_ENCODED_ACCOUNT_ID_SIZE,
+			bp_rococo::MAX_UNREWARDED_RELAYER_ENTRIES_AT_INBOUND_LANE as _,
+			bp_rococo::MAX_UNCONFIRMED_MESSAGES_AT_INBOUND_LANE as _,
+		)
+		.unwrap_or(u32::MAX);
+		pallet_bridge_messages::ensure_able_to_receive_confirmation::<Weights>(
+			bp_rococo::max_extrinsic_size(),
+			bp_rococo::max_extrinsic_weight(),
+			max_incoming_inbound_lane_data_proof_size,
+			bp_rococo::MAX_UNREWARDED_RELAYER_ENTRIES_AT_INBOUND_LANE,
+			bp_rococo::MAX_UNCONFIRMED_MESSAGES_AT_INBOUND_LANE,
+		);
+	}
+
+	#[test]
+	fn ensure_rococo_tx_extra_bytes_constant_is_correct() {
+		// **NOTE**: this test checks that we're computing transaction fee (for bridged chain, which, in
+		// case of Rococo<>Wococo, means any chain) on-chain properly. If this assert fails:
+		//
+		// 1) just fix the `TX_EXTRA_BYTES` constant to actual (or sightly rounded up) value;
+		// 2) (only if it has changed significantly (> x2 times)) ping the bridges team.
```

Comment:
```suggestion
		// 2) (only if it has changed significantly (> x2 times)) ping the bridges team (see the CODEOWNERS file if you're unsure who to ping)
```

This is probably gonna be a long line, so wrap it accordingly :P